projects/scripts/adi_board.tcl

```diff
@@ -297,7 +297,7 @@ proc ad_disconnect {p_name_1 p_name_2} {
 #  device clock
 #
 
-proc ad_xcvrcon {u_xcvr a_xcvr a_jesd {lane_map {}} {link_clk {}} {device_clk {}}} {
+proc ad_xcvrcon {u_xcvr a_xcvr a_jesd {lane_map {}} {link_clk {}} {device_clk {}} {num_of_max_lanes -1}} {
```

Comment:
Please add description of the num_of_max_lanes  parameter above in the comment section.